app/services/report_importer.rb

```diff
@@ -95,7 +83,11 @@ def import_log_messages
     end
   end
 
-  def inspect_report
+  def report_status
+    ConfigReportStatusCalculator.new(:counters => raw['status']).calculate
```

Comment:
When moving this to `config_report_importer` email tests from are raising an error (which is why I initially thought of moving the emailing tests to `config_report_importer_test`. I could instead return an empty hash, or move the email tests to `config_report_import_test`. Advise, please